Add known issues to Teams App caching page (#10548)
MicrosoftDocs/msteams-docs: msteams-platform/tabs/how-to/app-caching.md

  * `readyToUnload`
  * `getConfig/getSettings`

> * App caching is not supported in the [new Teams client](https://learn.microsoft.com/microsoftteams/platform/resources/teams-updates#known-issues)

Reviewer: ```suggestion
* App caching isn't supported in the [new Teams client](../resources/teams-updates.md#known-issues).
```
